cluster-autoscaler/FAQ.md

```diff
@@ -88,7 +88,15 @@ Cluster Autoscaler decreases the size of the cluster when some nodes are consist
   * are not run on the node by default, *
   * don't have a [pod disruption budget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#how-disruption-budgets-work) set or their PDB is too restrictive (since CA 0.6).
 * Pods that are not backed by a controller object (so not created by deployment, replica set, job, stateful set etc). *
-* Pods with local storage. *
+* Pods with local storage. *  
+Unless the pod has the following annotation set:
```

Comment:
Including the new info in this place unfortunately breaks the list which looks pretty confusing.

Sadly this point already has an asterisk that other points also use, so we can't piggyback on it.

I'd either:
* Add a second asterisk and include the information below the whole list, like we do for `cluster-autoscaler.kubernetes.io/safe-to-evict`.
* Shorten the information - use in-line code tags instead of the blocks, remove the clarification about using only one volume - if you say "a comma-separated list" I think it's pretty widely understood that a single item is also valid. Add the shortened info to the list in a sub-bullet - like we do for kube-system pods above.

The second one seems slightly better as the asterisks might get confusing.

Reply:
Thank you for the suggestion. The spacing for the last part does look different from the bullet. 

> Add a second asterisk and include the information below the whole list, like we do for cluster-autoscaler.kubernetes.io/safe-to-evict.

I kind of like this idea because we'd be able to write:
```
"cluster-autoscaler.kubernetes.io/safe-to-evict-local-volumes": "volume-1,volume-2"
```
as opposed to `
"cluster-autoscaler.kubernetes.io/safe-to-evict-local-volumes": "volume-1,volume-2"`. First option is easier to catch the eye because it is a block. 

We can get over the asterisk problem with something like this:

> 
> * Pods with local storage. * # 
> ...
>
> <sup>*</sup>Unless the pod has the following annotation (supported in CA 1.0.3 or later):
> ```
> "cluster-autoscaler.kubernetes.io/safe-to-evict": "true"
> ```
> __Or__
> 
> <sup>#</sup>Unless the pod has an annotation like:
> ```
> "cluster-autoscaler.kubernetes.io/safe-to-evict-local-volumes": "volume-1,volume-2"
> ```
> where `volume-1` and `volume-2` are safe to evict local storage volumes. You can also specify only one volume without any comma like this:
> ```
> "cluster-autoscaler.kubernetes.io/safe-to-evict-local-volumes": "volume-1"
> ```
> 
> __Or__ you have...

> Shorten the information - use in-line code tags instead of the blocks, remove the clarification about using only one volume - if you say "a comma-separated list" I think it's pretty widely understood that a single item is also valid.

I think this would be less confusing at the cost of being less visible.

Reply:
> if you say "a comma-separated list" I think it's pretty widely understood that a single item is also valid.

True. I just wanted to be explicit here :sweat_smile:

Reply:
P.S. I tried something and I think it should solve the formatting problem without us having to do 1 or 2 above. Just adding a few spaces to the line aligns it nicely with the bullet. Here's what I mean:
![image](https://user-images.githubusercontent.com/34534103/230541607-c37573a1-ccae-4082-a4f1-eafecf947a6f.png)

Reply:
I will try this and push. Let me know what you think.

Reply:
Added. https://github.com/kubernetes/autoscaler/pull/5594/files#r1160432848